v2/artemis/Dockerfile

```diff
@@ -10,20 +10,17 @@ WORKDIR /opt/artemis
 ## by version number. If this build starts breaking, the very likely cause is a
 ## 404 on the download. It can be fixed by upgrading to the latest Artemis or
 ## by changing the download URL.
-RUN apt-get update && \
-  apt-get install -y --no-install-recommends \
-    ca-certificates \
+RUN apk update && \
+  apk add \
     curl \
-    libaio1 \
-    xmlstarlet &&\ 
-  rm -rf /var/lib/apt/lists/* && \
+    openjdk11-jre \
+    xmlstarlet && \
```

Comment:
Switching from Debian to Alpine means package manager changes from `apt` to `apk`.